oat-sa/extension-tao-qtiprint
# Fix/BSA-162/remove second checkbox from Match interaction

Related to: https://oat-sa.atlassian.net/browse/BSA-162

Removed second checkbox in `label:before`

How to test:

1. create an item with Match interaction
2. create a test with this item
3. create a booklet from this test
4. check that in pfd and in the preview that only one checkbox displayed
